addons/reload/functions/fnc_displayAmmo.sqf

```diff
@@ -76,10 +76,14 @@ if (_muzzle == _weapon) then {
 } else {
     _showNumber = true;
 
+    _ammo = _target ammo _muzzle;
+    if (_ammo == 0) then { _magazine = ""; };
+
     _count = if (_magazine != "") then {
         {_x == _magazine} count (magazines _target + [_magazine])
     } else {
-        {_x in getArray (configFile >> "CfgWeapons" >> _weapon >> _muzzle >> "Magazines")} count magazines _target
+        private _compatibleMagazines =  [configFile >> "CfgWeapons" >> _weapon >> _muzzle] call CBA_fnc_compatibleMagazines;
+        {_x in _compatibleMagazines} count magazines _target
```

Comment:
What about the 3 round gl mags?

Reply:
This doesn't really change how they were handled
Maybe if magazine count > 1 then show the ammo bars like a normal magazine?